jerry-ext/debugger/debugger-tcp.c

```diff
@@ -144,6 +144,26 @@ jerryx_debugger_tcp_close (jerry_debugger_transport_header_t *header_p) /**< tcp
 
   jerryx_debugger_transport_tcp_t *tcp_p = (jerryx_debugger_transport_tcp_t *) header_p;
 
+  /* Waiting the debug client close the tcp connection first */
+  for (;;)
```

Comment:
use `while(true)`

The problem is this loop might never ends.

Reply:
Yeap, any suggestion? post here looking for suggestion

Reply:
> use `while(true)`
> 
> The problem is this loop might never ends.

maybe we doesn't need consider that, as when debugger stopping, means the debug client always closing the socket?